app/helpers/remote_execution_helper.rb

```diff
@@ -18,7 +18,7 @@ def host_counter(label, count)
   end
 
   def template_invocation_status(task, parent_task)
-    return(parent_task.result == 'cancelled' ? _('cancelled') : 'N/A') if task.nil?
+    return(parent_task.result == 'cancelled' ? _('cancelled') : _('Awaiting start')) if task.nil?
```

Comment:
Should a translation be added in a separate PR?

Reply:
Yes, strings are extracted for translation separately.